automobile/simple-vehicles.md

```diff
@@ -0,0 +1,75 @@
+## Simple Vehicles
+
+### CarSimple
+
+For each model of [Car](#car), a 'simple' PROTO is present too. These simplified
+kinematic PROTO models are not based on a `Robot` node but on a `Solid` node, it is
+therefore not possible to add sensors or control them. They are made to represent
+non-moving parked vehicles or to be moved using a Supervisor because they are much
+faster to simulate than the normal PROTO models.
+
+```
+ PROTO CarSimple {
+   field       SFVec3f    translation             0 0.4 0
+   field       SFRotation rotation                0 1 0 0
+   field       SFColor    color                   0.0 0.25 0.65
+   field       MFColor    recognitionColors       [ 0.0 0.25 0.65, 0.1 0.1 0.1 ]
+   field       MFString   plate                   "textures/plate.jpg"
+   field       SFString   name                    "vehicle"
+   field       SFBool     wheelBoundingObject     FALSE
+ }
+ ```
+
+#### CarSimple Field Summary
+
+- `recognitionColors`: If not empty, this vehicle may be recognized by any Camera
+device with recognition capability (i.e. with a Recognition node).
+- `wheelBoundingObject`: Allows to enable the physical geometry of the wheels.
```

Comment:
If I am not wrong you should always use the 'allows' verb in the sense 'allows someone to...'.